plugin/references.py

```diff
@@ -37,7 +37,10 @@ def run(self, edit: sublime.Edit, event: Optional[dict] = None, point: Optional[
         session = self.best_session(self.capability)
         file_path = self.view.file_name()
         if session and file_path:
-            pos = get_position(self.view, event, point)
+            tmp_pos = get_position(self.view, event, point)
+            if tmp_pos is None:
+                return
+            pos = tmp_pos
```

Comment:
Had to do this "tmp" stuff because python is weird. Or lambdas rather since they don't bind the variables so the type will revert to possibly being `None`

Reply:
can we just put?
```python
if session and file_path:
  pos = get_position(self.view, event, point)
  if pos is None:
      return
  window = self.view.window()
```